angular.json

```
@@ -245,12 +264,13 @@
               "projects/cardiovascular-health/src/assets"
             ],
             "styles": [
-              "projects/cardiovascular-health/src/styles.scss"
+              "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css",
+              "projects/showcase/src/styles.scss"
```

this should not be loading the showcase stylesheet.

```suggestion
              "./dist/ngx-charts-on-fhir/themes/default.scss",
              "projects/cardiovascular-health/src/styles.scss"
```